torch/nn/utils/clip_grad.py

```diff
@@ -31,7 +31,7 @@ def clip_grad_norm_(parameters, max_norm, norm_type=2):
     clip_coef = max_norm / (total_norm + 1e-6)
     if clip_coef < 1:
         for p in parameters:
-            p.grad.data.mul_(clip_coef.item())
+            p.grad.data.mul_(float(clip_coef))
```

Comment:
this doesn't look right either, and is an unreviewed change, so has to be backed out.

Reply:
Caused by D7784132